test_scripts/API/RAI/043_ATF_P_steeringWheelLocation_LEFT_response_transfered_fake_param_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,226 @@
+-- In InitHMI_OnReady HMI replies with parameters: 
+-- steeringWeelLocation = LEFT to UI.GetCapabilities
+-- fake parameter from VR.GetCapabilities: vrCapabilities
+--
+-- 2. Performed steps
+-- Register new application.
+--
+-- Expected result:
+-- SDL->mobile: RegisterAppInterface_response steeringWeelLocation is provided equal to LEFT
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForRAI = require('user_modules/shared_testcases/testCasesForRAI')
+local events = require("events")
+local mobile_session = require('mobile_session')
+
+--[[ Local functions ]]
+
+--[[@text_field - sets parameters of structure textFields
+--! @ used in UI.GetCapabilities
+--! @parameters: name, characterSet, width, rows
+--]]
+local function text_field(name, characterSet, width, rows)
+    return 
+    { name = name, characterSet = characterSet or "TYPE2SET", width = width or 500, rows = rows or 1 }
+end
+
+--[[@text_field - sets parameters of structure imageFields
+--! @ used in UI.GetCapabilities
+--! @parameters: name, width
+--]]  
+local function image_field(name, width)
+  return
+    { name = name,
+      imageTypeSupported =
+      {
+        "GRAPHIC_BMP",
+        "GRAPHIC_JPEG",
+        "GRAPHIC_PNG"
+      },
+      imageResolution =
+      {
+        resolutionWidth = width or 64,
+        resolutionHeight = 64
+      }
+    }
+end
+
+--[[ General Precondition before ATF start ]]
+commonPreconditions:BackupFile("hmi_capabilities.json")
+testCasesForRAI.write_data_steeringWheelLocation("RIGHT")
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--TODO(istoimenova): shall be removed when issue: "ATF does not stop HB timers by closing session and connection" is fixed
+config.defaultProtocolVersion = 2
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+	testCasesForRAI.InitHMI_onReady_without_UI_GetCapabilities(self)
+	local event = events.Event()
+	event.level = 2
+	event.matches = function(_, data) return data.method == "UI.GetCapabilities" end
+
+	EXPECT_HMIEVENT(event, "UI.GetCapabilities")
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, "UI.GetCapabilities", "SUCCESS", {
+			vrCapabilities = { "TEXT" }, --fake parameter
+			hmiCapabilities = 
+      {
+				navigation = false,
+				phoneCall = true,
+				steeringWheelLocation = "RIGHT"
```

Comment:
@istoimenova
According test case description: In InitHMI_OnReady HMI replies with parameter steeringWeelLocation = LEFT to UI.GetCapabilities
In script it is steeringWheelLocation = "RIGHT"
Please check.